documentation/src/main/asciidoc/topics/con_xsite_auto_offline.adoc

```diff
@@ -63,15 +69,17 @@ include::yaml/xsite_backup_number_failures.yaml[]
 You can also specify how long to wait before taking sites offline when backup operations fail.
 If a backup request succeeds before the wait time runs out, {brandname} does not take the site offline.
 
-[TIP]
-====
-Set a negative or zero value for number of failures to use only a minimum time to wait to take backup locations offline.
+Within the range of one or two minutes is generally a suitable time to wait before automatically taking backup locations offline.
+If the wait period is too short then backup locations go offline too soon.
+You then need to bring clusters back online and perform state transfer operations to ensure data is in sync between the clusters.
+
+You can also set a negative or zero value for number of failures to use only a minimum time to wait to take backup locations offline, for example:
 
 [source,xml,options="nowrap",subs=attributes+]
 ----
-<take-offline after-failures="-1" min-wait="10000"/>
+<take-offline after-failures="-1"
```

Comment:
It's not clear how this is different from `<take-offline after-failures="1"` /cc @pruivo

Reply:
it is not.
`-1` just means Infinispan will not count the failures but you need 1 failure to start the timer :)